fizzbuzz.py

```diff
@@ -0,0 +1,53 @@
+class Fizzbuzz:
+    def __init__(self, fb_list = None) -> None:
```

Comment:
I've set the optional to none, then have it check if its none and if so, set it to an empty list. If I initialize the list as an optional parameter, it means every Fizzbuzz object will share that same list. By initializing it in the method, it ensures each object has its own list.